lib/job/repeat-every.ts

```diff
@@ -7,9 +7,14 @@ import { Job } from '.';
  * @param interval repeat every X
  * @param options options to use for job
  */
-export const repeatEvery = function(this: Job, interval: string | number, options: any = {}): Job {
+export const repeatEvery = function(this: Job, interval: string | number, options: { timezone?: string; startDate?: Date | number; endDate?: Date | number; skipDays?: string; skipImmediate?: boolean } = {}): Job {
   this.attrs.repeatInterval = interval;
   this.attrs.repeatTimezone = options.timezone ? options.timezone : null;
+  // Following options are added to handle start day
+  // and cases like run job every x days (skip some days)
+  this.attrs.startDate = options.startDate ?? null;
```

Comment:
maybe it's intended, just wanted to point it out: startDate can either be Date | number or undefined..and here undefined is "converted" to null. I usually try to keep things either undefined or null, but not both for "same" purposes of having things "not defined". Or was here any specific reason to set startDate to "null" by default then? (Ohterwise I think just remove ?? null would be fine too)

Reply:
I don't have a preference for null vs undefined.

I reuse the code from the original PR.
I kept the existing style.
for example `timezone` is checked for undefined and set to null.
I'm fine either way.